x-pack/legacy/plugins/alerting/server/lib/create_execution_handler.test.ts

```diff
@@ -29,7 +30,12 @@ beforeEach(() => jest.resetAllMocks());
 
 test('calls executeAction per selected action', async () => {
   const executionHandler = createExecutionHandler(createExecutionHandlerParams);
-  await executionHandler('default', {}, {});
+  await executionHandler({
+    actionGroup: 'default',
```

Comment:
oh good, sending the group as well!  I think I had thought about that as another missing piece before, forgot to jot that down anywhere though.

oh wait, but I don't see it below, *explicitly*, in the `transformActionParams()` usage below, so maybe it isn't being sent.  Could do it in a follow-on PR if it seems like something we'd want; seems like it could be.

Reply:
Good point. Thinking of it, we already know the group when defining the action (for a given alert group). I can't think of a use case that group could be dynamic?